xsgen/brightlite.py

```diff
@@ -27,14 +30,30 @@ def write(self, libs, dirname):
         for mat, matlib in libs.items():
             if isinstance(mat, int):
                 fname = str(nucname.zzaaam(mat))
+            #elif mat == 'fuel':
+            #    fname = mat
             else:
-                fname = mat
+                continue
             lines = [row + "   " + "   ".join(map(str, matlib[row]))
                      for row in rownames]
+            trans_matrix = {}
+            i = 0
+            while i < len(matlib['material']):
+                for temp_nuc in matlib['material'][i].comp:
+                    nuc_name = str(nucname.name(temp_nuc))
+                    try:
+                        trans_matrix[nuc_name].append(matlib['material'][i].comp[temp_nuc])
+                    except KeyError:
+                        if matlib['material'][i].comp[temp_nuc] > 1.E-4:
```

Comment:
There shouldn't be any magic threshold numbers. These should be user-configurable.